multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1369,11 +1371,14 @@ def _fit(
                 ".* in the `DataLoader` init to improve performance.*",
             )
             warnings.filterwarnings("ignore", "Checkpoint directory .* exists and is not empty.")
+            training_start = time.time()
```

Comment:
This only counts the lightning trainer's fitting time. Should we calculate predictor's fitting time?

Reply:
Sure, we can do that. So how about I put the `training_start = time.time()` in the very beginning of fit()? like here https://github.com/awslabs/autogluon/blob/master/multimodal/src/autogluon/multimodal/predictor.py#L537

And then put the end_time at here? 
https://github.com/awslabs/autogluon/blob/master/multimodal/src/autogluon/multimodal/predictor.py#L729

Reply:
Sure.

Reply:
Done